api/nakadi-event-bus-api.yaml

```diff
@@ -1210,6 +1219,22 @@ definitions:
       - type
       - schema
 
+  EventTypeStatistics:
+    type: object
+    description: |
+      Operational statistics for an EventType. This data is generated by Nakadi based on the runtime and
+      might be used to guide changes in internal parameters.
+
+    properties:
+      expected_write_rate:
```

Comment:
This is not finished.
I don't see the need to offer such flexibility at this time, but obviously further descriptions about this are important and will be added.